src/router.tsx

```diff
@@ -1,67 +1,109 @@
-import { createBrowserRouter } from "react-router-dom";
-import App from "./App";
-import Home from "./pages/home";
-import Main from "./pages/main";
-import Financial from "./pages/products/financial";
-import FinancialId from "./pages/products/financial/[financialId]";
-import SignIn from "./pages/signin";
-import SignUp from "./pages/signup";
-import User from "./pages/user";
-import Edit from "./pages/user/Edit";
-import MyCart from "./pages/user/MyCart";
-import WishList from "./pages/user/WishList";
-import OrderList from "./pages/user/OrderList";
+import { createBrowserRouter } from 'react-router-dom';
+import App from './App';
+import Home from './pages/home';
+import Main from './pages/main';
+import Financial from './pages/products/financial';
+import FinancialId from './pages/products/financial/[financialId]';
+import SignIn from './pages/signin';
+import SignUp from './pages/signup';
+import User from './pages/user';
+import Edit from './pages/user/Edit';
+import MyCart from './pages/user/MyCart';
+import WishList from './pages/user/WishList';
+import OrderList from './pages/user/OrderList';
+import { ProtectedRoute } from './pages/ProtectedRoute';
+import { SuccessRoute } from './pages/ProtectedRoute';
 
 const router = createBrowserRouter([
   {
-    path: "/",
+    path: '/',
     element: <App />,
     children: [
       {
         index: true,
         element: <Home />,
       },
       {
-        path: "main",
-        element: <Main />,
+        path: 'main',
+        element: (
+          <ProtectedRoute>
+            <Main />
+          </ProtectedRoute>
```

Comment:
```suggestion
          {/* <ProtectedRoute> /*}
            <Main />
          {/* </ProtectedRoute> /*}
```
아직 [인증기능 구현 -> 모든 개발자가 같은 인증 기능으로 테스트]
하는 단계가 아니라서 
지금은 주석 처리해두는게 좋을것 같습니다.